core/themes/basis/template.php

```diff
@@ -4,6 +4,17 @@
  * Basis preprocess functions and theme function overrides.
  */
 
+/**
+ * Implements hook_css_alter().
+ */
+function basis_css_alter(&$css) {
+  // Remove Basis' `/css/component/menu-dropdown.css` if using a custom
+  // breakpoint.
+  if (config_get('system.core', 'menu_breakpoint') != 'default') {
+    unset($css['core/themes/basis/css/component/menu-dropdown.css']);
```

Comment:
This is edge-case handling, but the path `core/themes/basis` might change if the user were to copy Basis into a contrib/custom theme directory. Could we revise this to:

```suggestion
    $path = backdrop_get_path('theme', 'basis');
    unset($css[$path . '/css/component/menu-dropdown.css']);
```